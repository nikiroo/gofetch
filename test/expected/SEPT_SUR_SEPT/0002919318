     L%27OFFICE DES ÉTRANGERS N%27EXÉCUTE PAS UNE INJONCTION DE    
                 L%27ONU DE LIBÉRER LA FAMILLE SERBE               

  [Belgique] 26-09-2018 17:00

  o News link: http://s.7s7.be/3479998
  o Source link: 


    Par: rédaction
    
    26/09/18  -  17h00  Source:  Belga    Theo Francken, secrétaire
    d'État à l'Asile et la Migration. © photo news.
    
    FAMILLE  SERBE  L'Office des étrangers a reçu une ordonnance du
    Comité  des droits de l'enfant de l'Onu pour libérer la famille
    serbe  enfermée  depuis  le  14  août  dernier  dans les unités
    familiales  du  centre  127bis  de  Steenokkerzeel,  mais il ne
    l'exécute  pas  car il ne reconnaît pas le pouvoir d'injonction
    de cet organisme onusien.
    
    L'information,  annoncée  par  Ecolo  dans un communiqué, a été
    confirmée à Belga par l'Office des étrangers.
    
    Les  avocats de cette famille avaient déposé une requête auprès
    du  Comité  des  droits  de  l'enfant des Nations unies dans le
    cadre de sa compétence de plaintes individuelles.
    
    "Cet  organe  de  l'ONU  est  l'instance  suprême  en charge du
    contrôle  de  la  mise  en œuvre par les États de la Convention
    des  Nations  unies  sur  les droits de l'enfant (CIDE). Depuis
    2011,  ce  comité  est  compétent  pour  recevoir  des plaintes
    individuelles  des  enfants  dont les droits ont été violés. La
    Belgique  a ratifié ce mécanisme de plaintes en 2014", souligne
    Ecolo.
    
    [  "Le  gouvernement  belge  doit  respecter  les  décisions de
    l'ONU, point" ]
    
    "Pourquoi  le  secrétaire d'État à l'Asile et la Migration Theo
    Francken  n'a-t-il  pas suivi l'ordonnance du Comité des droits
    de  l'enfant  de  l'ONU",  demande  le  député  fédéral  Benoît
    Hellings.
    
    A  ses  yeux, cette décision est d'autant plus incompréhensible
    que  le  Premier  ministre  Charles  Michel est actuellement au
    siège   de   l'ONU   pour  y  défendre  les  principes  et  les
    institution des Nations unies.
    
    "Le  gouvernement  belge doit respecter les décisions de l'ONU,
    point", conclut le député d'opposition.



