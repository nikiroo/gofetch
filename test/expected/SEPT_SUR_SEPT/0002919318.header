0L'Office des étrangers n'exécute pas une injonction de l'ONU de libérer la famille serbe	null/SEPT_SUR_SEPT/0002919318		70
i  [Belgique] 26-09-2018 17:00
i
i    L'Office  des  étrangers  a  reçu  une ordonnance du Comité des
i    droits  de  l'enfant  de  l'Onu  pour  libérer la famille serbe
i    enfermée depuis le 14 ...
i
