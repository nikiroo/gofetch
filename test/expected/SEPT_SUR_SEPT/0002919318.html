<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>L'Office des étrangers n'exécute pas une injonction de l'ONU de libérer la famille serbe</h2>
	<div class='details'>([Belgique] 26-09-2018 17:00)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0002919318</a></li>
        <li>News link: <a href='http://s.7s7.be/3479998'>http://s.7s7.be/3479998</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 17h00 Source: Belga    Theo Francken, secrétaire d&#x27;État à l&#x27;Asile et la Migration. © photo news.<br/><br/>FAMILLE SERBE L&#x27;Office des étrangers a reçu une ordonnance du Comité des droits de l&#x27;enfant de l&#x27;Onu pour libérer la famille serbe enfermée depuis le 14 août dernier dans les unités familiales du centre 127bis de Steenokkerzeel, mais il ne l&#x27;exécute pas car il ne reconnaît pas le pouvoir d&#x27;injonction de cet organisme onusien.<br/><br/>L&#x27;information, annoncée par Ecolo dans un communiqué, a été confirmée à Belga par l&#x27;Office des étrangers.<br/><br/>Les avocats de cette famille avaient déposé une requête auprès du Comité des droits de l&#x27;enfant des Nations unies dans le cadre de sa compétence de plaintes individuelles.<br/><br/>&quot;Cet organe de l&#x27;ONU est l&#x27;instance suprême en charge du contrôle de la mise en œuvre par les États de la Convention des Nations unies sur les droits de l&#x27;enfant (CIDE). Depuis 2011, ce comité est compétent pour recevoir des plaintes individuelles des enfants dont les droits ont été violés. La Belgique a ratifié ce mécanisme de plaintes en 2014&quot;, souligne Ecolo.<br/><br/><h3>&quot;Le gouvernement belge doit respecter les décisions de l&#x27;ONU, point&quot;</h3><br/><br/>&quot;Pourquoi le secrétaire d&#x27;État à l&#x27;Asile et la Migration Theo Francken n&#x27;a-t-il pas suivi l&#x27;ordonnance du Comité des droits de l&#x27;enfant de l&#x27;ONU&quot;, demande le député fédéral Benoît Hellings.<br/><br/>A ses yeux, cette décision est d&#x27;autant plus incompréhensible que le Premier ministre Charles Michel est actuellement au siège de l&#x27;ONU pour y défendre les principes et les institution des Nations unies.<br/><br/>&quot;Le gouvernement belge doit respecter les décisions de l&#x27;ONU, point&quot;, conclut le député d&#x27;opposition.
	</div>
<hr/>
</div>
</body>
